docs/dev_guides/changing_splink/contributing_to_docs.md

```diff
@@ -16,21 +16,27 @@ Once you've finished updating Splink documentation we ask that you run our spell
 
 ## Spellchecking docs
 
-When updating Splink documentation, we ask that you run our spellchecker before submitting a pull request. This is to help ensure quality and consistency across the documentation. Please note, the spellchecker _only works on markdown files_ and currently only works on systems which support `Homebrew` package manager. Instructions for other operating systems will be released later.
+When updating Splink documentation, we ask that you run our spellchecker before submitting a pull request. This is to help ensure quality and consistency across the documentation. If for whatever reason you can't run the spellchecker on your system, please don't let this prevent you from contributing to the documentation. Please note, the spellchecker _only works on markdown files_.
+
+If you are a Mac user with the `Homebrew` package manager installed, the script below will automatically install
+the required system dependency, `aspell`.
+If you've created your development environment [using conda](./development_quickstart.md), `aspell` will have been installed as part of that
+process.
+Instructions for installing `aspell` through other means may be added here in the future.
 
 To run the spellchecker on either a single markdown file or folder of markdown files, you can use the following script:
```

Comment:
Suggestion to change this text to support the minor edit to the syntax in the code block following it

```suggestion
To run the spellchecker on either a single markdown file or folder of markdown files, you can run the following bash script:
```